public/app/features/storage/StoragePage.tsx

```diff
@@ -175,6 +263,12 @@ export default function StoragePage(props: Props) {
                 Dashboard
               </LinkButton>
             )}
+
+            {canAddFolder && (
+              <FileUpload accept={fileFormats} onFileUpload={onFileUpload} showFileName={true} key={fileUploadKey}>
+                Upload
+              </FileUpload>
+            )}
             {canAddFolder && <Button onClick={() => setIsAddingNewFolder(true)}>New Folder</Button>}
```

Comment:
```suggestion
            {canAddFolder && (<>
              <FileUpload accept={fileFormats} onFileUpload={onFileUpload} showFileName={true} key={fileUploadKey}>
                Upload
              </FileUpload>
              <Button onClick={() => setIsAddingNewFolder(true)}>New Folder</Button>
              </>
             }
```

^^ but with formatting :)

Since both things have the same condition, we can group them together.  the `<>...</>` structure lets you add multiple items when it expects one chile